design/setup.md

```diff
@@ -102,3 +103,29 @@ and the CLI is attached to it. A cluster config looks like this:
 
     [cluster]
     name= "<cluster_name>"
+
+## Automatically installing default plugins
+
+Finally, and unless the `--no-plugin` option is passed, the CLI will attempt to install "dcos-core-cli"
+and "dcos-enterprise-cli" plugins.
+
+These plugins download URLs are retrieved through Cosmos, where they are registered as packages.
+
+### dcos-core-cli
+
+The [core plugin](https://github.com/dcos/dcos-core-cli) contains subcommands such as marathon, job, node,
+package, service, task.
+
+When the core plugin can't be installed (eg. insufficient Cosmos permission or air-gapped environment),
+it then fallbacks to installing it from the DC/OS CLI binary itself, which bundles a core plugin.
```

Comment:
"fallbacks" -> "falls back".

We should make clear that which version of the core CLI it has bundled will be dependent on the version of the wrapper CLI you happen to be using and that it won't check the version of the core it has against the DC/OS cluster.